cvat-canvas/src/typescript/sliceHandler.ts

```diff
@@ -554,8 +554,12 @@ export class SliceHandlerImpl implements SliceHandler {
             }], 'slice');
 
             this.objectSelector.enable(([state]) => {
-                this.objectSelector.disable();
-                initializeWithContour(state);
+                if (state) {
+                    this.objectSelector.disable();
+                    initializeWithContour(state);
+                } else {
+                    this.release();
+                }
```

Comment:
Would not it be more correct just do nothing if `state` is undefined? 

I believe, e.g. changing of z layer or deleting another object after enabling slice might be, in general, a legal operation.

Additionally, if we want to release slice handler, it should be done from notify method as for other staff, to avoid one more implicit way of cancellation